pkg/infoschema/cluster.go

```diff
@@ -112,23 +118,26 @@ func init() {
 		cols = append(cols, memTableCols...)
 		tableNameToColumns[clusterMemTableName] = cols
 	}
+	for memTableName, clusterMemTableName := range memTableToDDLOwnerClusterTables {
+		memTableToDDLOwnerClusterTablesWithLowerCase[strings.ToLower(memTableName)] = strings.ToLower(clusterMemTableName)
+	}
 }
 
 // IsClusterTableByName used to check whether the table is a cluster memory table.
 // Export for PhysicalTableScan.ExplainID
 func IsClusterTableByName(dbName, tableName string) bool {
-	dbName = strings.ToUpper(dbName)
+	intest.Assert(dbName == strings.ToLower(dbName))
```

Comment:
`strings.ToLower` will be called first. Maybe use `AssertFunc` instead.

Reply:
Done